templates/form.html

```diff
@@ -52,6 +53,27 @@ <h1 class = "title" >Username Availability Checker</h1>
                 </center>
             </div>
         </div>
+        <div id="usernames">
+            <p class="textualbk"> Need a username? </p>
+            <form id="usernames-form">
+                <label for="min_length">Min Length</label>
+                <input type="number" min="4" max="14" id="min_length" value="6" required> <br/>
```

Comment:
Probably cleaner to not have a space between `> <br/>` because I _believe_ that will add a space to the end of the line:

```suggestion
                <input type="number" min="4" max="14" id="min_length" value="6" required><br/>
```

Also applies to inputs below.

Reply:
I think it's also cleaner to do all the line breaks and spacing in CSS so it's more easy to change in the future (though I think it'd be fine making that a separate issue if you're unable to get it working quickly).